README.md

```diff
@@ -575,6 +575,7 @@ A GitHub conflict is when people make changes to the same area or line in a file
   - [Ayush Luthra](https://github.com/Ayushluthra2001)
   - [Ayush Mishra](https://github.com/ayush-sleeping)
   - [Ayush Sarode](https://github.com/AyushSarode)
+  - [Ayush Rai](https://github.com/ayush1rai)
```

Comment:
I believe your name should be one line above (`S` comes first then `R`).
Let us know if you need any help with that. 🙂